recipes/rsem/build.sh

```diff
@@ -34,6 +37,55 @@ rsem-tbam2gbam \
 rsem_perl_utils.pm \
 "
 
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+
+########################################################
+### Prepare Samtools/Htslib 
+########################################################
+
+sed -i.bak 's/^CPPFLAGS =$//g' samtools-1.3/Makefile
```

Comment:
Upon further inspection rsem requires some header files that are not in the conda package. I run make to get the headers, but use the bioconda package for the actual installed versions.